compiler/rustc_resolve/src/imports.rs

```diff
@@ -175,7 +175,7 @@ pub(crate) struct ImportData<'a> {
     pub module_path: Vec<Segment>,
     /// The resolution of `module_path`.
     pub imported_module: Cell<Option<ModuleOrUniformRoot<'a>>>,
-    pub vis: Cell<Option<ty::Visibility>>,
+    pub vis: Option<ty::Visibility>,
```

Comment:
```suggestion
    pub vis: ty::Visibility,
```
This visibility should never be `None` now.
(That means `expect_vis` can also be removed.)